### PR TITLE
WOE/WOC all 38 Role cards + Sleep Cursed Faerie

### PR DESCRIPTION
**Pull request by Simisays**
somethings are not yet clear, whether the Roles are enchant creature or enchant permanent

### CODE REVIEW

File: forge-gui/res/tokenscripts/role_wicked.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Wicked Role
+ManaCost:no cost
+Types:Enchantment Aura Role
+K:Enchant creature
+A:SP$ Attach | Cost$ 0 | ValidTgts$ Creature | AILogic$ Pump
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddPower$ 1 | AddToughness$ 1 | Description$ Enchanted creature gets +1/+1 
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Execute$ TrigDamage | TriggerDescription$ When this Aura is put into a graveyard, each opponent loses 1 life.
```
Comment (Hanmac) [inline forge-gui/res/tokenscripts/role_wicked.txt:7]: `Destination$ Graveyard` ?

File: forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ManaCost:1 R
+Types:Creature Human Rogue
+PT:1/1
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm | TriggerDescription$ When CARDNAME enters the battlefield, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBLoot,DBTreasure,DBToken
+SVar:DBLoot:DB$ Discard | Mode$ TgtChoose | SubAbility$ DBDraw | SpellDescription$ Discard a card, then draw a card.
+SVar:DBDraw:DB$ Draw
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | SpellDescription$ Create a Wicked Role token attached to target creature you control.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt:11]: AI Logic Curse there?

File: forge-gui/res/tokenscripts/role_cursed.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Cursed Role
+ManaCost:no cost
+Types:Enchantment Aura Role
+K:Enchant creature
+A:SP$ Attach | Cost$ 0 | ValidTgts$ Creature | AILogic$ Curse
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | SetPower$ 1 | SetToughness$ 1 | Description$ Enchanted creature is 1/1
+Oracle:Enchanted creature is 1/1
```
Comment (Hanmac) [inline forge-gui/res/tokenscripts/role_cursed.txt:7]: The Cursed Role should probably have some kind of "SacMe" Flag for the AI

File: forge-gui/res/cardsfolder/upcoming/cursed_courtier.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Cursed Courtier
+ManaCost:2 W
+Types:Creature Human Noble
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a Cursed Role token attached to it. (Enchanted creature is 1/1.)
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_cursed | AttachedTo$ Self
+DeckHas:Ability$Token|LifeGain & Type$Aura|Enchantment|Rolesure token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cursed_courtier.txt]: typo

File: forge-gui/res/cardsfolder/upcoming/faunsbane_troll.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Faunsbane Troll
+ManaCost:2 B G
+Types:Creature Troll
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a Monster Role token attached to it. (Enchanted creature gets +1/+1 and has trample.)
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_monster | AttachedTo$ Self
+A:AB$ Fight | Defined$ Self | Cost$ Sac<1/Aura.Attached> | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | ReplaceDyingDefined$ Targeted | SorcerySpeed$ True | SpellDescription$ CARDNAME fights target creature you don't control. If that creature would die this turn, exile it instead. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/faunsbane_troll.txt:7]: wrong tgts

File: forge-gui/res/cardsfolder/upcoming/living_lectern.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Living Lectern
+ManaCost:1 U
+Types:Artifact Creature Construct
+PT:0/4
+A:AB$ Draw | Cost$ 1 Sac<1/CARDNAME> | SubAbility$ DBToken | SpellDescription$ Draw a card. Create a Sorcerer Role token attached to up to one other target creature you control. Activate only as a sorcery. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has “Whenever this creature attacks, scry 1.”)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/living_lectern.txt:5]: SorcerySpeed

File: forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Name:Twisted Fealty
+ManaCost:2 R
+Types:Sorcery
+A:SP$ GainControl |ValidTgts$ Creature | TgtPrompt$ Select target creature | LoseControl$ EOT | Untap$ True | AddKWs$ Haste | SubAbility$ DBToken | SpellDescription$Gain control of target creature until end of turn. Untap that creature. It gains haste until end of turn.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Curse | TgtPrompt$ Select up to one target creature you control | SpellDescription$ Create a Wicked Role token attached to up to one target creature. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1. When this Aura is put into a graveyard, each opponent loses 1 life.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt:5]: Targeted too broad in this case

**Comment by Hanmac** [comment]
The Name of the Role token is probably something like "Monster Token" or "Monster Role Token"

but that can be changed later

File: forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt [hunk 8]
```diff
@@ -2,6 +2,6 @@ Name:Twisted Fealty
...
-SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | TargetMin$ 0 | TargetMax$ 1 | AILogic$ Curse | TgtPrompt$ Select up to one target creature you control | SpellDescription$ Create a Wicked Role token attached to up to one target creature. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1. When this Aura is put into a graveyard, each opponent loses 1 life.)
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature |  AILogic$ Curse | TargetMin$ 0 | TargetMax$ 1 | SpellDescription$ Create a Wicked Role token attached to up to one target creature. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1. When this Aura is put into a graveyard, each opponent loses 1 life.)
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt:5]: The Wicked Aura gives +1/+1 so it isn't a Curse

File: forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt (hunk 7 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt]: Does it need `AttachedTo$ ThisTargetedCard ?`

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/twisted_fealty.txt]: yes

File: forge-gui/res/tokenscripts/role_cursed.txt [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Name:Cursed Role
```
Comment (Hanmac) [inline forge-gui/res/tokenscripts/role_cursed.txt:1]: The names of them are very most likely `Cursed Token`

Comment (Simisays) [reply forge-gui/res/tokenscripts/role_cursed.txt:1]: I think just `cursed `? 
image

Comment (Hanmac) [reply forge-gui/res/tokenscripts/role_cursed.txt:1]: yeah, but each Token needs to have "Token" at the end i think?

but yeah you might do it with just "Cursed" for now

File: forge-gui/res/cardsfolder/upcoming/become_brutes.txt [hunk 10]
```diff
@@ -0,0 +1,8 @@
+Name:Become Brutes
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Pump | TargetMin$ 1 | KW$ Haste | TargetMax$ 2 | ValidTgts$ Creature | TgtPrompt$ Select one or two target creatures | SubAbility$ DBRepeat | RememberTargets$ True | SpellDescription$ One or two target creatures each gain haste until end of turn.
+SVar:DBRepeat:DB$ RepeatEach | RepeatCards$ Creature.YouCtrl+IsRemembered | Zone$ Battlefield | RepeatSubAbility$ DBToken | ChangeZoneTable$ True | SpellDescription$ For each of those creatures, create a Monster Role token attached to it. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/become_brutes.txt:5]: repeat with Targeted

File: forge-gui/res/cardsfolder/upcoming/syr_armont_the_redeemer.txt [hunk 11]
```diff
@@ -0,0 +1,10 @@
+Name:Syr Armont, the Redeemer
+ManaCost:3 G W
+Types:Legendary Creature Human Knight
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME, the Redeemer enters the battlefield, create a Monster Role token attached to another target creature you control. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/syr_armont_the_redeemer.txt:5]: fix desc

File: forge-gui/res/cardsfolder/upcoming/syr_armont_the_redeemer.txt [hunk 12]
```diff
@@ -0,0 +1,10 @@
...
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_monster | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl+StrictlyOther | TgtPrompt$ Select another target creature you control
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/syr_armont_the_redeemer.txt:6]: don't need Strictly with Targeting

**Comment by Hanmac** [comment]
just to be clear, i will handle the unattach of the Roles later when we get more clarification on what happens when multiple roles are attached at the same time.

if they are attached + unattached in Sequence
or if they are attached at once and the others unattached at once
or if they are just unattached as SBA

File: forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt [hunk 13]
```diff
@@ -0,0 +1,9 @@
...
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | RememberObjects$ Targeted | Execute$ TrigFight | ConditionDefined$ Remembered | ConditionPresent$ Card | TriggerDescription$ When you do, that creature fights up to one target creature you don't control. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample. Creatures that fight each deal damage equal to their power to the other.)
+SVar:TrigFight:DB$ Fight | Defined$ Remembered | ValidTgts$ Creature.YouDontCtrl | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature you don't control | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt:7]: not needed

File: forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | RememberObjects$ Targeted | Execute$ TrigFight | ConditionDefined$ Remembered | ConditionPresent$ Card | TriggerDescription$ When you do, that creature fights up to one target creature you don't control. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample. Creatures that fight each deal damage equal to their power to the other.)
+SVar:TrigFight:DB$ Fight | Defined$ Remembered | ValidTgts$ Creature.YouDontCtrl | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target creature you don't control | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt:6]: wrong defined

File: forge-gui/res/cardsfolder/upcoming/cursed_courtier.txt [hunk 15]
```diff
@@ -2,6 +2,7 @@ Name:Cursed Courtier
 ManaCost:2 W
 Types:Creature Human Noble
 PT:3/3
+K:LifeGain
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cursed_courtier.txt:5]: ??

File: forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt [hunk 16]
```diff
@@ -0,0 +1,8 @@
+Name:Curse of the Werefox
+ManaCost:2 G
+Types:Sorcery
+A:SP$ Token | TokenAmount$ 1 | TokenScript$ role_monster | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | RememberTokens$ True | SubAbility$ DBImmediateTrigger | SpellDescription$ Create a Monster Role token attached to target creature you control. When you do, that creature fights up to one target creature you don't control. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample. Creatures that fight each deal damage equal to their power to the other.)
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | RememberObjects$ Targeted | Execute$ TrigFight | ConditionDefined$ Remembered | ConditionPresent$ Card | TriggerDescription$ When you do, that creature fights up to one target creature you don't control. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has trample. Creatures that fight each deal damage equal to their power to the other.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/curse_of_the_werefox.txt:5]: needs cleanup?

File: forge-gui/res/cardsfolder/upcoming/besotted_knight_betroth_the_beast.txt [hunk 17]
```diff
@@ -0,0 +1,15 @@
...
+DeckHas:Ability$Token & Type$Aura|Enchantment|Role
+AlternateMode:Adventure
+Oracle:
+
+ALTERNATE
+
+Name:Betroth the Beast
+ManaCost:G
+Types:Sorcery Adventure
+A:SP$ Token | TokenAmount$ 1 | TokenScript$ role_royal | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | AILogic$ Curse | TgtPrompt$ Select target creature you control | SpellDescription$ Create a Royal Role token attached to target creature you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/besotted_knight_betroth_the_beast.txt:14]: why Curse?

File: forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt [hunk 18]
```diff
@@ -0,0 +1,13 @@
...
+PT:1/1
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm | TriggerDescription$ When CARDNAME enters the battlefield, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBLoot,DBTreasure,DBToken
+SVar:DBLoot:DB$ Discard | Mode$ TgtChoose | SubAbility$ DBDraw | SpellDescription$ Discard a card, then draw a card.
+SVar:DBDraw:DB$ Draw
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | AILogic$ Curse | TgtPrompt$ Select target creature you control | SpellDescription$ Create a Wicked Role token attached to target creature you control.
+DeckHas:Ability$Token|Discard & Type$Aura|Enchantment|Role|Treasure
+Oracle:Haste\nWhen Charming Scoundrel enters the battlefield, choose one -\n* Discard a card, then draw a card.\n* Create a Treasure token.\n* Create a Wicked Role token attached to target creature you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt:13]: Wrong symbols

File: forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt [hunk 19]
```diff
@@ -0,0 +1,13 @@
...
+ManaCost:1 R
+Types:Creature Human Rogue
+PT:1/1
+K:Haste
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm | TriggerDescription$ When CARDNAME enters the battlefield, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBLoot,DBTreasure,DBToken
+SVar:DBLoot:DB$ Discard | Mode$ TgtChoose | SubAbility$ DBDraw | SpellDescription$ Discard a card, then draw a card.
+SVar:DBDraw:DB$ Draw
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | AILogic$ Curse | TgtPrompt$ Select target creature you control | SpellDescription$ Create a Wicked Role token attached to target creature you control.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/charming_scoundrel.txt:11]: the Wicked Role isn't AILogic$ Curse

File: forge-gui/res/cardsfolder/upcoming/embereth_veteran.txt [hunk 20]
```diff
@@ -0,0 +1,7 @@
+Name:Embereth Veteran
+ManaCost:R
+Types:Creature Human Knight
+PT:2/1
+A:AB$ Token | Cost$ 1 Sac<1/CARDNAME> | TokenAmount$ 1 | TokenScript$ role_young_hero | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl+Other | AILogic$ Curse | TgtPrompt$ Select another target creature you control | SpellDescription$ Create a Young Hero Role token attached to another target creature. (If you control another Role on it, put that one into the graveyard. Enchanted creature has "Whenever this creature attacks, if its toughness is 3 or less, put a +1/+1 counter on it.")
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/embereth_veteran.txt:5]: AILogic$ Pump

File: forge-gui/res/cardsfolder/upcoming/living_lectern.txt [hunk 21]
```diff
@@ -0,0 +1,8 @@
+Name:Living Lectern
+ManaCost:1 U
+Types:Artifact Creature Construct
+PT:0/4
+A:AB$ Draw | SorcerySpeed$ True | Cost$ 1 Sac<1/CARDNAME> | SubAbility$ DBToken | SpellDescription$ Draw a card. Create a Sorcerer Role token attached to up to one other target creature you control. Activate only as a sorcery. (If you control another Role on it, put that one into the graveyard. Enchanted creature gets +1/+1 and has “Whenever this creature attacks, scry 1.”)
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TargetMin$ 0 | TargetMax$ 1 | TokenScript$ role_sorcerer | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl+Other | AILogic$ Curse | TgtPrompt$ Select up to one other target creature you control
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/living_lectern.txt:6]: AILogic$ Pump

File: forge-gui/res/cardsfolder/upcoming/vantress_transmuter.txt [hunk 22]
```diff
@@ -0,0 +1,16 @@
...
+AlternateMode:Adventure
+Oracle:
+
+ALTERNATE
+
+Name:Croaking Curse
+ManaCost:1 U
+Types:Sorcery Adventure
+A:SP$ Tap |ValidTgts$ Creature | SubAbility$ DBToken | SpellDescription$ Tap target creature. Create a Cursed Role token attached to it. (Enchanted creature is 1/1.)
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ role_cursed | TokenOwner$ You | AttachedTo$ Targeted
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/vantress_transmuter.txt:15]: AILogic$ Curse

File: forge-gui/res/cardsfolder/upcoming/living_lectern.txt (hunk 21 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/living_lectern.txt:6]: just none

File: forge-gui/res/cardsfolder/upcoming/conceited_witch_price_of_beauty.txt [hunk 23]
```diff
@@ -0,0 +1,16 @@
...
+DeckHas:Ability$Token & Type$Aura|Enchantment|Role
+AlternateMode:Adventure
+Oracle:Menace (This creature can't be blocked except by two or more creatures.)
+
+ALTERNATE
+
+Name:Price of Beauty
+ManaCost:B
+Types:Sorcery Adventure
+A:SP$ Token | TokenAmount$ 1 | TokenScript$ role_wicked | TokenOwner$ You | AttachedTo$ Targeted | ValidTgts$ Creature.YouCtrl | AILogic$ Pump | TgtPrompt$ Select target creature you control | SpellDescription$ Create a Wicked Role token attached to target creature you control. (Then exile this card. You may cast the creature later from exile.)
```
[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/conceited_witch_price_of_beauty.txt:15]: well Pump is kind of the default assumption for now (same reason we don't tag every Pump$ effect)

**Comment by Simisays** [comment]
Removed the AIHints for the token lines

**Review by tool4ever — APPROVED** [review]
nice work

**Comment by kevlahnota** [comment]
> somethings are not yet clear, whether the Roles are enchant creature or enchant permanent

I thought the release notes stated that roles are enchant creature?
IMG_20230829_063609

**Review by Hanmac — APPROVED** [review]

**Comment by Simisays** [comment]
> > somethings are not yet clear, whether the Roles are enchant creature or enchant permanent
> 
> I thought the release notes stated that roles are enchant creature?
> IMG_20230829_063609
> 

Yeah this was an old comment from back when this mechanic was just leaked. You're right that all the roles have enchant creature

**Pull request merged**